src/main/java/mate/jdbc/util/TestUtil.java

```diff
@@ -0,0 +1,110 @@
+package mate.jdbc.util;
+
+import java.util.Scanner;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
+public class TestUtil {
```

Comment:
Actually, it's a bad practice to mix your production code with test-related code. If you do such comprehensive testing, it's better to create separate test classes and use junit, mockito and so on

Reply:
I know. But the README file says that we should test in the main (

Reply:
Consider moving it to the Main class than, instead of separate TestUtil